src/shared/glean/gleandeprecated.h

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef GLEANDEPRECATED_H
+#define GLEANDEPRECATED_H
+
+#include <QObject>
+#include <QVariant>
+
+class GleanDeprecated final : public QObject {
```

Comment:
Please add a comment to this class on why it is deprecated and that it should not be used in new apps.